src/pages/learn/index.mdx

```diff
@@ -52,8 +56,52 @@ Could produce the following JSON result:
 
 ```json
 {
-  "me": {
-    "name": "Luke Skywalker"
+  "data": {
+    "me": {
+      "name": "Luke Skywalker"
+    }
   }
 }
 ```
+
+With even a simple query, we can see some of the key features that make GraphQL so powerful. The client can make queries to the API that mirror the structure of the data that they need and then receive just that data in the expected shape with a single request—and without having to worry about what underlying data sources provided it.
+
+## Evolve your API without versioning
+
+Client requirements change over time and GraphQL allows your API to evolve in response to those needs and without the overhead of managing different API versions. For example, if a new feature calls for more specific name values to be available, then the `User` type could be updated as follows:
+
+```graphql
+type User {
+  fullName: String
+  nickname: String
+  name: String @deprecated(reason: "Use `fullName`.")
+}
+```
+
+GraphQL will continue to provide data as expected for the deprecated `name` field until it's determined that clients have updated their existing queries and the older field can be safely removed.
```

Comment:
This edit clarifies that releasing an update to all of the clients is not sufficient - we also need to ensure that no existing (non-updated) clients remain - e.g. mobile apps from years ago. It also makes it clear that GraphQL itself does not make any determination that the field is no longer used.

```suggestion
Client tooling will encourage developers to use the new fields and remove usage of the deprecated `name` field. The field can be removed once it is determined it is no longer used; in the meantime GraphQL will continue to provide its data as expected.
```